### ISSUE TITLE
Windows Narrator in "scan mode" does not read ComboBox "dropdown" items after focusing input and opening "dropdown"

### ISSUE THREAD

**Issue by hoovercj** [OfficeDev/office-ui-fabric-react#6441]
### Please provide a reproduction of the bug in a codepen:

[Office Fabric ComboBox](https://developer.microsoft.com/en-us/fabric#/components/ComboBox
) 
[codepen](https://codepen.io/anon/pen/LJaLqx)

### Bug Report 
 
Windows Narrator in "scan mode" does not read ComboBox "dropdown" items after focusing input and opening "dropdown".

- __Package version(s)__: All
- __Browser and OS versions__: Windows 10 + Edge
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: High
 
Products/sites affected: Microsoft products
 
#### Describe the issue:
 
* Go to the codepen or ComboBox docs page in Microsoft Edge
* Activate Windows Narrator in "standard" mode (scan mode turned off)
* Tab to the example captioned "Basic uncontrolled example (allowFreeform: T, AutoComplete: T):
* Tab to the "Set Focus" button
* Press enter
* Note: The focus has moved to the input and the dropdown has expanded
* Press Caps+Space to activate scan mode in Windows Narrator
* Press Down Arrow Key button
 
#### Actual behavior:
 
Windows Narrator reads the "Set Focus" button and the next elements after that button.
 
#### Expected behavior:
 
According to accessibility test vendors, the dropdown elements should be read.

I can't find anything in the spec that suggests this is necessary, and I can't find any examples to support or refute their claims. [This](https://w3c.github.io/aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html) is the closest I can find via w3c.

#### Questions

I've marked this as high priority because we've marked this bug as "external" and would like to receive quick feedback that either this is by design and won't be changed (with links to spec or other resources we can use to defend the position) or that is considered a bug on your backlog so we can pass that on to our vendors.

**Comment by dzearing** [comment]
@jspurlin Do you have any feedback here?

**Comment by micahgodbolt** [comment]
@aneeshack4 is this another issue related to #6422.

**Comment by aneeshack4** [comment]
@micahgodbolt At first glance it seems like let me dig into this a little bit more though

**Comment by aneeshack4** [comment]
@micahgodbolt When I ran Narrator on this, in both Scan Mode & non-Scan Mode, pressing the down arrow while in the dropdown causes Narrator to say "Unable to move Narrator cursor" and the focus stays on the selected element in the dropdown... so I would say this isn't just a scan mode issue. But this issue report says for them, Narrator moves out of the dropdown and onto the Set Focus button and the elements after it.

**Comment by hoovercj** [comment]
I have double checked that this behaves as I initially described in Microsoft Edge 42.17134.1.0.

This contrasts slightly with the [Combobox with Listbox Popup Examples](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/combobox/aria1.1pattern/listbox-combo.html) provided by w3.org.

In those examples, pressing "down" will navigate into the dropdown in scan mode. I assume this is because the dropdown is placed directly after the input element in the DOM.

The relevant parts from the bug report logged by our vendors:

> **Repro Steps:**
> <[see initial issue text](https://github.com/OfficeDev/office-ui-fabric-react/issues/6441#issue-362579266)>
> 
> **Actual Result:**
> In scan mode, Narrator focus doesn't move to 'list item' available under filter while navigating over the screen
> 
> **Expected Result:** 
> Narrator focus should move to 'list item' available under Filter while navigating over the screen in scan mode as well.
> 
> **Note:**
> This issue also exists with Chrome + JAWS(Touch Cursor mode).
> 
> **MAS Reference:**
> [MAS 4.2.10 - Tracking of Focus and Selection Attributes (33J)](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7bb24201d1-2276-4471-8183-e1e5353031ad%7d)

**Comment by eveleens** [comment]
Any traction on this one? We have the same bug. 

> Actual Result:
> Narrator is not reading state as Expanded/Collapsed for combo box.
> 
> Expected result:
> Narrator should read state as Expanded/Collapsed for combo box.
> 
> Note:
> Issue does not exist with Chrome + Jaws. Jaws reading correct state as expand or collapsed properly.
> 
> MAS Reference:
> MAS 4.2.9 - Execution of Available Actions

**Comment by eveleens** [comment]
Any traction on this one? We have the same bug for ComboBox - in scan mode, pushing the down arrow doesn't focus on the elements.

**Comment by jspurlin** [comment]
I think this would be a good one to bring up to the Narrator team. This seems to be an issue with scan mode.

**Comment by aneeshack4** [comment]
@eveleens I think @jspurlin is right - we've come across a lot of similar issues with our components in Scan Mode. Next step here is to contact the Narrator team - their workflow for issues like this is to post them on the internal stack overflow. I'll post there and send y'all the link on Teams. I'll close this Github issue since the SO one will now track it.
